tensorflow/java/src/gen/perl/tftypes.pl

```diff
@@ -0,0 +1,138 @@
+#!/usr/bin/perl
```

Comment:
Having this in perl is a bit worrying, given that most of the TensorFlow maintainers are not perl users :)

So this script will have to be rewritten at some point in Python, but I guess in the mean time having the generated files checked in works out well.